index.html

```diff
@@ -303,6 +303,20 @@ <h3 class="name">Rémi JARA</h3>
                 <a href="https://www.linkedin.com/in/remijara/" target="_blank"><i class="fa-brands fa-linkedin-in"></i></a>
             </div>
         </div>
+        
+                <div class="profile">
+            <div class="pfp"><img src="https://telegra.ph/file/8a257ffcbce41b06c315f.jpg" alt="User Image"></div>
```

Comment:
the image link is invalid

Reply:
Please Can you dm me on dc

I don't know how to get the Github Img link